### ISSUE TITLE
Dataset with multi-view videos 

### ISSUE THREAD

**Issue by ayesha-ishaq** [OpenGVLab/InternVL#624]
Hi, thank you for your work!
Can I insert placeholder tokens for my dataset for multiple videos (essentially, a video with frames representing multiple views at each timestamp)? I want the model to see the multiple views together at each time step. I see that \<video> placeholder is replaced with 'Frame{}: \<image>', and in multi-image dataset \<image>, placeholders are used the same number of times. Can these two be combined to allow the model to focus on multiple view frames across time?

Thank you!

**Comment by czczup** [comment]
Yes, you can do this, just notice the context length of LLM and the number of visual tokens.
